packages/code-studio/src/dashboard/linker/LinkerOverlayContent.tsx

```diff
@@ -121,15 +154,14 @@ export class LinkerOverlayContent extends Component {
           return null;
         }
       })
-      .filter(item => item != null);
+      .filter(item => item != null) as VisibleLink[];
 
     return (
       <div className="linker-overlay">
         <svg>
           {visibleLinks.map(({ x1, y1, x2, y2, id, className }) => (
             <LinkerLink
               className={className}
-              disabled={disabled}
```

Comment:
Why was this deleted with no replacement?

Reply:
The prop didn't exist on LinkerLink at all. It just passes a `.disabled` class on the className now.